src/cpu/test_data.zig

```diff
@@ -0,0 +1,10141 @@
+/// Seed: 0x51133 (slime, if you squint)
```

Comment:
how was this file generated? would be nice to have the script in the repo :)

Reply:
uhh, I hijacked the startup method of a decompiled 1.8 client - i could separate it into a standalone program, but it'd still be java